cmd/empire/factories.go

```diff
@@ -418,13 +380,34 @@ func newHBReporter(key, env string) (reporter.Reporter, error) {
 	return append(reporter.MultiReporter{}, reporter.NewLogReporter(), r), nil
 }
 
+// Stats =======================
+
+func newStats(c *Context) (stats.Stats, error) {
+	if addr := c.String(FlagDogStatsd); addr != "" {
+		return newDataDogStats(addr)
+	}
+
+	return stats.Null, nil
+}
+
+func newDataDogStats(addr string) (stats.Stats, error) {
```

Comment:
It might be worth at least having a datadog & plain statsd implementation at first.  I think the only difference would be removing tags?